elodie.py

```diff
@@ -181,8 +183,22 @@ def update_location(media, file_path, location_name):
             print(('{"source":"%s",' % file_path,
                 '"error_msg":"Failed to update location"}'))
             sys.exit(1)
-    return True
 
+        config = load_config()
+        if ('Location' in config) and (config['Location'].getboolean('write_location')):
```

Comment:
Instead of doing the check here I think it belongs in `media.set_location`. This way when you set the location on a file it will always add the tags.